azure-pipelines.yml

```diff
@@ -1,4 +1,16 @@
 jobs:
+- job: BiDiScan
+  pool:
+    vmImage: 'ubuntu-18.04'
+
+  steps:
+  - checkout: self
+    fetchDepth: 32
+
+  - script: |
+      git fetch origin 8.0
```

Comment:
Did you try without `fetchDepth: 32` ?
`git fetch origin 8.0` may be not required and Azure might be faster. I'm not sure but it's worth checking.

Reply:
It's tricky. 'fetchDepth: 32' asks `git` to download only 32 last commits from a repo history. It makes less data to be downloaded for client. From the other side, it requires a git server to prepare and compress a repo history just for your call so it makes a git server slower. We use 'fetchDepth: 32' mainly because it takes less disk space on Azure VM.

Reply:
For BiDi you can experiment what is faster (because we have a lot of free disk space) but I don't expect a huge difference.

Reply:
@inikep, without fetchDepth and 'git fetch' the job runs for about 90s longer.

Reply:
Thanks for checking